src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Shapes/EllipseAlignment2542_2547.xaml

```diff
@@ -0,0 +1,75 @@
+﻿<Page
+	x:Class="UITests.Windows_UI_Xaml_Shapes.EllipseAlignment2542_2547"
+	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	mc:Ignorable="d">
+
+	<StackPanel Spacing="10" Margin="10">
+		<TextBlock FontSize="15">
+			This is an illustration of Github bugs
+			<Hyperlink NavigateUri="https://github.com/unoplatform/uno/issues/2542">#2542 (iOS)</Hyperlink>
+			and
+			<Hyperlink NavigateUri="https://github.com/unoplatform/uno/issues/2547">#2547 (Android)</Hyperlink>
+		</TextBlock>
+
+		<TextBlock Foreground="BlueViolet" FontSize="14">
+			--> You should see a white circle, containing an outlined red circle. THEY SHOULD BE PERFECTLY CENTERED.
+		</TextBlock>
+
+		<Border Background="Blue" Height="100">
+			<Grid Height="74"
+			  Width="74"
+			  VerticalAlignment="Center"
+			  HorizontalAlignment="Center">
+
+				<!-- [iOS] BUG #2542 : This White ellipse isn't visible on iOS with latest Uno.UI dev -->
+				<Ellipse VerticalAlignment="Stretch"
+					 HorizontalAlignment="Stretch"
+					 Fill="White" />
+
+				<!-- [Android] BUG #2547 : This Red stroked ellipse isn't correctly centered on Android with latest Uno.UI dev -->
```

Comment:
Can be removed ;)

Reply:
Why?  That's an illustration of the bug...

Reply:
@carldebilly but this PR fix those bugs no? Do you still want to keep those comments in that case? (since you already have a good description above)